Explorer/Assets/DCL/AvatarRendering/AvatarShape/UnityInterface/MainPlayerAvatarBaseProxy.cs

```diff
@@ -0,0 +1,15 @@
+namespace DCL.AvatarRendering.AvatarShape.UnityInterface
+{
+    public class MainPlayerAvatarBaseProxy
+    {
+        public AvatarBase AvatarBase { get; private set; }
+
+        public bool Configured { get; private set; }
```

Comment:
Same as Proxy<T>

Reply:
Tackled the needed refactor unifying the "proxy" usages into a new `ObjectProxy<T>` at [this commit](https://github.com/decentraland/unity-explorer/pull/340/commits/e58d6c8346f2450f1f8bfa137c2e610704d62991)